### ISSUE TITLE
Can not build on Windows

### ISSUE THREAD

**Issue by sinhpham** [mitsuhiko/redis-rs#101]
I got this error when trying to build on Windows:
```
error: no associated item named `Unix` found for type `connection::ConnectionInfo` in the curre
nt scope
ConnectionInfo::Unix(_) => false,
```

It was building fine previously (version 0.5.3 I think), seems like the new version introduced a new dependency on Unix stream? If so, can we use conditional compilation to make it build on Windows again?

**Comment by badboy** [comment]
See http://mitsuhiko.github.io/redis-rs/redis/#optional-features

**Comment by phsym** [comment]
Looks like the issue will be fixed by #102

**Comment by badboy** [comment]
Yup, fixed.
